packages/patternfly-4/react-core/src/components/AlertGroup/AlertGroup.tsx

```diff
@@ -0,0 +1,57 @@
+import styles from '@patternfly/react-styles/css/components/AlertGroup/alert-group';
+import { css } from '@patternfly/react-styles';
+import { canUseDOM } from 'exenv';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { Omit } from '../../helpers/typeUtils';
+
+export interface AlertGroupProps extends Omit<React.HTMLProps<HTMLUListElement>, 'className'> {
+  /** Additional classes added to the AlertGroup */
+  className?: string;
+  /** Alerts to be rendered in the AlertGroup */
+  children?: React.ReactNode;
+  /** Toast notifications are positioned at the top right corner of the viewport */
+  isToast?: boolean;
+  /** Determine where the alert is appended to */
+  appendTo?: HTMLElement | (() => HTMLElement);
```

Comment:
looks like appendTo is only invoked as a function below, so this should only take an HTMLElement or the code is updated below with something like:
```
getModalRoot = (appendTo: HTMLElement | (() => HTMLElement)) => {
    if (typeof appendTo === 'function') {
      return appendTo();
    }
    return appendTo || document.body;
  };
```

Also appendTo should allow the user to append the modal to any element not just document.body, so i think this needs to be changed a bit. To put it all together, something like:

```
constructor(props) {
    super(props);
    this.el = document.createElement('div');
  }

componentDidMount() {
  getModalRoot().appendChild(this.el);
}

componentWillUnmount() {
  getModalRoot(). removeChild(this.el);
}

getModalRoot = (appendTo: HTMLElement | (() => HTMLElement)) => {
    if (typeof appendTo === 'function') {
      return appendTo();
    }
    return appendTo || document.body;
  };

render() {
  const alertGroup = ...
    return ReactDOM.createPortal(
      alertGroup,
      this.el,
    );
  }
```

Reply:
Might be good to create an example / test for it as well